src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -140,6 +141,12 @@ protected List<Action> retrieveActions(SCMHead head,
                 .findAny()
                 .ifPresent(action -> result.add(new PrimaryInstanceMetadataAction()));
         }
+
+        if (head instanceof BitbucketPullRequestSCMHead) {
+            MinimalPullRequest pullRequest = ((BitbucketPullRequestSCMHead) head).getPullRequest();
+            result.add(new ObjectMetadataAction(pullRequest.getTitle(), pullRequest.getDescription(), null));
```

Comment:
Is there any value in providing the `ObjectUrl` as well?

Reply:
Thanks for raising. Happy to add it in as well.